### PR TITLE
[MIG] 10.0 Porting hr_contract_default_trial_length

### PR DESCRIPTION
**Pull request by jlaloux**
Module works fine. No need special adaptations for migration to 10.0

### CODE REVIEW

File: hr_contract_default_trial_length/README.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Installation
```
Comment (feketemihai) [inline hr_contract_default_trial_length/README.rst:12]: @jlaloux Thanks for your contribution, can you please add the Try me on Runbot section, and add yourself in the Contributors list...

File: hr_contract_default_trial_length/__manifest__.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 ###############################################################################
```
Comment (feketemihai) [inline hr_contract_default_trial_length/__manifest__.py]: @jlaloux Can you adapt to have small headers...

File: hr_contract_default_trial_length/models/hr_contract.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 #
 ###############################################################################
```
Comment (feketemihai) [inline hr_contract_default_trial_length/models/hr_contract.py]: @jlaloux Can you adapt to have small headers...

File: hr_contract_default_trial_length/models/hr_contract_type.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 #    along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 ###############################################################################
```
Comment (feketemihai) [inline hr_contract_default_trial_length/models/hr_contract_type.py]: @jlaloux Can you adapt to have small headers...

**Review by feketemihai — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**